algorithms/src/std_algorithms/numeric/Kokkos_ExclusiveScan.hpp

```diff
@@ -57,6 +57,30 @@ namespace Kokkos {
 namespace Experimental {
 namespace Impl {
 
+template <class ExeSpace, class IndexType, class ValueType, class FirstFrom,
+          class FirstDest>
+struct ExclusiveScanDefaultFunctorNoInit {
```

Comment:
@masterleinad I have one more suggestion here: the name `ExclusiveScanDefaultFunctorNoInit` is misleading in my view, beucase the functor still needs an init value. I would suggest renaming this to `ExclusiveScanDefaultFunctorForKnownNeutralElement`

Reply:
[301b879](https://github.com/kokkos/kokkos/pull/4886/commits/301b87974484b0818b82af18800fb67b819cc9bf)